tests/BaseTest.php

```diff
@@ -7,22 +7,47 @@
 
 abstract class BaseTest extends TestCase
 {
-    public function onlyGuzzle6()
+    protected function onlyGuzzle5()
     {
-        $version = ClientInterface::VERSION;
-        if ('6' !== $version[0]) {
+        if ($this->getGuzzleVersion() !== 5) {
+            $this->markTestSkipped('Guzzle 5 only');
+        }
+    }
+
+    protected function onlyGuzzle6()
+    {
+        if ($this->getGuzzleVersion() !== 6) {
             $this->markTestSkipped('Guzzle 6 only');
         }
     }
 
-    public function onlyGuzzle5()
+    protected function atLeastGuzzle6()
```

Comment:
hm... Im not sure about naming. This will need to change when Guzzle 8 comes out. 

How about `onlyGuzzle6and7`? Or `onlyGuzzle6` and `onlyGuzzle7`?
Or maybe even `requireGuzzleVersion(... $versions)`?

Reply:
I am okay with `onlyGuzzle6and7`, might as well be explicit